docs/extend/data.md

```diff
@@ -144,15 +125,18 @@ You can also add [relationships](https://laravel.com/docs/5.7/eloquent-relations
         ->belongsToMany('role', 'App\Role', 'role_user', 'user_id', 'role_id')
 ```
 
-You may pass a closure as the final argument to customize the relationship object further:
+Those 4 should cover the majority of relations, but sometimes, finer-grained customization is needed (e.g. `morphMany`, `morphToMany`, and `morphedByMany`). ANY valid Eloquent relationship can be supported by the relationship method:
```

Comment:
"can be supported" doesn't sound like the kind of phrasing we want in a documentation :innocent: 

```suggestion
Those 4 should cover the majority of relations, but sometimes, finer-grained customization is needed (e.g. `morphMany`, `morphToMany`, and `morphedByMany`). ANY valid Eloquent relationship is supported by the `relationship` method:
```